optaplanner-spring-integration/optaplanner-spring-boot-autoconfigure/src/main/java/org/optaplanner/spring/boot/autoconfigure/config/BenchmarkProperties.java

```diff
@@ -14,17 +14,35 @@
  * limitations under the License.
  */
 
-package org.optaplanner.spring.boot.autoconfigure;
+package org.optaplanner.spring.boot.autoconfigure.config;
 
 import org.springframework.boot.context.properties.NestedConfigurationProperty;
 
 public class BenchmarkProperties {
+
+    public static final String DEFAULT_SOLVER_BENCHMARK_CONFIG_URL = "solverBenchmarkConfig.xml";
+    public static final String DEFAULT_BENCHMARK_RESULT_DIRECTORY = "target/benchmarks";
```

Comment:
We can move these here, that does not affect backwards compatibility towards the application.properties contracts.